book/src/passive-stake-delegation-and-rewards.md

```diff
@@ -0,0 +1,149 @@
+# Stake Delegation and Reward
+
+This design proposal focuses on the software architecture for the on-chain
+voting and staking programs.  Incentives for staking is covered in [staking
+rewardsd](staking-rewards.md).
+
+The current architecture requires an active vote for each delegated stake from
+the validator, and therefore does not scale to allow for replicator clients
+automatically delegate their rewards.
+ 
+The design proposes a new set of programs for voting and stake delegation, The
+proposed programs allow many stakes to passively earn rewards with a single
+validator vote without permission from the validator.
+
+## Current Design Problems
+
+In the current design each staker creates their own VoteState, and assigns a
+**delegate** in the VoteState that can submit votes.  Since the validator has to
+actively vote for each stake delegated to it, it can censor stakes by not voting
+for them.
+
+The number of votes is equal to the number of stakers, and not the number of
+validators.  Replicator clients are expected to automatically delegate their
+rewards, and therefore the number of stakes is expected to be large compared to
+the number of validators in a long running cluster.
+
+## Proposed changes to the current design.
+
+The general idea is that instead of the staker, the validator will own the
+VoteState program. In this proposal the VoteState program is there to track
+validator votes, count validator generated credits and to provide any
+additional validator specific state.  The VoteState program is not aware of any
+stakes delegated to it, and has no staking weight.
+
+The rewards generated are proportional to the amount of lamports staked.  In
+this proposal stake state is stored as part of the RewardsState program. This
```

Comment:
Why the association with the Rewards program? How about a new Stake program with StakeState?

Reply:
Then the Rewards program would go away.  Since claiming the reward updates the claimed_rewards counter.

Reply:
Easy enough to propose deleting the Rewards program